test/spec_lint.rb

```diff
@@ -611,34 +611,12 @@ def assert_lint(*args)
     assert_lint nil, ''.dup
     assert_lint 1, ''.dup
   end
-
```

Comment:
Is there a reason we would want to remove this spec if we aren't removing hijacking?

Reply:
Nope, we should leave it in, my bad.